templates/sync-catalog-deployment.yaml

```
@@ -1,3 +1,4 @@
+{{ $clientEnabled := ternary true false (or (and (ne (.Values.client.enabled | toString) "-") .Values.client.enabled) (and (eq (.Values.client.enabled | toString) "-") .Values.global.enabled)) }}
```

```suggestion
{{- $clientEnabled := (or (and (ne (.Values.client.enabled | toString) "-") .Values.client.enabled) (and (eq (.Values.client.enabled | toString) "-") .Values.global.enabled)) }}
```

1. I think we can assign directly to the variable
2. Should have the whitespace chomping